woocommerce/woocommerce
# add support for custom E2E container name

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds support for configuring a customer container name prefix. It updates the core container prefix from `woocommerce` to `woocommerce_e2e`

Closes #28012 .

### How to test the changes in this Pull Request:

1. Ensure you have run `npm run docker:down` before switching to this branch
2. Switch to branch and run `npm run docker:up`
3. Note container name changes
4. `npm run docker:down`
5. Edit `tests/e2e/config/default.json` and change the appName entry `woocommerce_{your-name}`
6. `docker:up`, `'test:e2e`, `docker:down`

### Other information:


### Changelog entry

N/A
